src/Diagnostics.ModelsAndUtils/ScriptUtilities/Utilities.cs

```diff
@@ -11,75 +11,85 @@ public class Utilities
         /// <summary>
         /// Creates a kusto query part with time filters
         /// </summary>
-        /// <param name="cxt"><see cref="OperationContext"/></param>
+        /// <param name="cxt"><see cref="OperationContext{TResource}"/></param>
         /// <param name="timeColumnName">Name of the Time Column</param>
         /// <returns>Kusto time filter query part</returns>
         /// <example> 
         /// This sample shows how to use <see cref="TimeFilterQuery"/> method.
         /// <code>
-        /// public string GetKustoQuery(OperationContext cxt)
+        /// public string GetKustoQuery(<![CDATA[OperationContext<App> cxt]]>)
         /// {
         ///     string timeFilterQueryPart = Utilities.TimeFilterQuery(cxt);
         ///     return $"SomeTableName | where {timeFilterQueryPart}";
         /// }
         /// </code>
         /// This will produce "SomeTableName | where <![CDATA[PreciseTimeStamp >= datetime(2018-04-04 04:10) and PreciseTimeStamp <= datetime(2018-04-05 04:10) ]]>"
         /// </example>
-        public static string TimeFilterQuery(OperationContext cxt, string timeColumnName = "PreciseTimeStamp")
+        public static string TimeFilterQuery(string startTime, string endTime, string timeColumnName = "PreciseTimeStamp")
         {
-            return $"{timeColumnName} >= datetime({cxt.StartTime}) and {timeColumnName} <= datetime({cxt.EndTime})";
+            return $"{timeColumnName} >= datetime({startTime}) and {timeColumnName} <= datetime({endTime})";
         }
 
         /// <summary>
         /// Creates a kusto query part for Tenant filters. This is a recommended way to use Tenant filters as this takes care of mega-stamps also.
         /// </summary>
-        /// <param name="cxt"><see cref="OperationContext"/></param>
+        /// <param name="cxt"><see cref="OperationContext{TResource}"/></param>
```

Comment:
Remove this to match IResource

Reply:
I need to correct the comments. I'll work on the documentation later.